configure

```diff
@@ -7866,7 +7867,8 @@ print_enabled_components(){
     struct_name=$2
     name=$3
     shift 3
-    echo "static const $struct_name * const $name[] = {" > $TMPH
+    echo "/* Automatically generated by configure - do not modify! */" > $TMPH
+    echo "static const $struct_name * const $name[] = {" >> $TMPH
     for c in $*; do
```

Comment:
Why?
This is none of our business :-)

Reply:
It just bothered me, I had just modified an auto-generated file without knowing it was auto-generated, and I wondered why it didn't have that usual disclaimer, so I decided to fix it. But I just remembered FFmpeg standards would have this be a separate commit, so I'll remove it 👍